README.md

```diff
@@ -46,48 +46,34 @@ This required re-writing the history of the repo. You **must** [pull and rebase
 If you want to contribute to the code, please fork the repository, make your changes, and create a pull request.
 
 ### Local setup
+Build the necessary services:
 
-Install the [PostgreSQL](https://www.postgresql.org/) database server. It's a useful idea to setup [passwordless authentication for local connections](https://www.postgresql.org/docs/current/static/auth-methods.html#AUTH-TRUST).
+    sudo docker-compose build
 
-You'll also need the psql and libxml development libraries. 
+Setup the database:
 
-```sh
-# Ubuntu/debian
-sudo apt-get install libpq-devel libxml2-dev libxslt1-dev python-dev
-# macOS
-brew install libxml2
-```
-
-You'll need python 2.7 and [virtualenv](https://virtualenv.pypa.io/en/stable/installation/).
-
-Clone this repo, and setup a virtualenv:
-
-    virtualenv --no-site-packages env
-    source env/bin/activate
-
-Install requirements:
-
-    pip install -r requirements.txt
+    sudo docker-compose run --rm web python setup_database.py
```

Comment:
Let's make it very clear that it sets up a dev database and shouldn't be used in production

```suggestion
    sudo docker-compose run --rm web python setup_dev_database.py
```

Reply:
Okay